supabase/migrations/20231014161224_fix_api_key_policy.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+  ALTER POLICY "allow apikey to select" ON "public"."apps" USING (is_allowed_capgkey(((current_setting('request.headers'::text, true))::json ->> 'capgkey'::text), '{all,write,upload,read}'::key_mode[], app_id));
```

Comment:
Is it possible to write this in more than one line? It's pretty hard to read everything in one single line

Reply:
I do not know, I have copied this code from supabase and have not changed a bit. The truth is that we have even longer lines in [base.sql](https://github.com/Cap-go/capgo/blob/2210c90ae8c9dd957914d658b37e7f6f5cf989a1/supabase/migrations/20230815171919_base.sql#L2247) so I would not risk breaking this